pkg/test/framework.go

```diff
@@ -57,6 +57,10 @@ type Framework struct {
 }
 
 func setup(kubeconfigPath, namespacedManPath *string) error {
+	namespace := ""
+	if *singleNamespace {
+		namespace = os.Getenv(TestNamespaceEnv)
```

Comment:
If `len(namespace) == 0` exit with error `TestNamespaceEnv` must be set.

Reply:
Not necessary. We default to the namespace specified in the kubeconfig in that case: https://github.com/operator-framework/operator-sdk/pull/770/files/b7ca022a01320d6b684df1d9e4cec56c86151440#diff-7ebc985f9067a44fc5791eeec5c424e2R84